src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/adapters/TableAdapter.test.js

```diff
@@ -68,6 +68,84 @@ test('Render data with schema', () => {
     expect(tableAdapter).toMatchSnapshot();
 });
 
+test('Render different kind of data with schema', () => {
```

Comment:
Can't we integrate that test in the above one? I've been trying to reduce the amount of snapshot tests lately, because they also have downsides if there are too many (nobody checks snapshots anymore, it is not completely clear what is being tested here, ...)